README.md

```diff
@@ -1606,9 +1631,6 @@ that successive retries would use wait times of 1, 2, 4, 8, and 16 seconds.
 
 
 #### With external configuration
-Note: the configuration of automatic retries via external configuration
-is currently supported only in the Go and Python SDKs.
-
```

Comment:
When this PR will be merged, all SDKs will already support retries, therefore this note won't be necessary.